docker-compose.yml

```diff
@@ -203,6 +203,7 @@ services:
       - METRICS_URL=http://metrics:1050
       - FHIR_URL=http://hearth:3447/fhir
       - APPLICATION_CONFIG_URL=http://config:2021/
+      - COUNTRY_CONFIG_URL=http://countryconfig:3040
```

Comment:
Added this to the `e2e` repo here: https://github.com/opencrvs/e2e/commit/80dc8efc89303c0129a1de69ce4ba6f731f4d2b6 :slightly_smiling_face: